tests/e2e/commerce/order/asset/test_sync_asset.py

```
@@ -0,0 +1,116 @@
+    # Must use this fixture for all tests to prevent api errors
+    asset_data = order_asset_factory()
+    orders = mpt_vendor.commerce.orders
+    assets = orders.assets(commerce_asset_draft_order_id)
+    asset = assets.create(asset_data)
+
+    yield asset
+
+    try:
+        assets.delete(asset.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to delete order asset: {getattr(error, 'title', str(error))}")  # noqa: WPS421
```

Ruff (0.14.8)

Unused `noqa` directive (unknown: `WPS421`)

Remove unused `noqa` directive

(RUF100)